source/core/parameters.cc

```diff
@@ -914,7 +918,7 @@ namespace Parameters
       tracer_order        = prm.get_integer("tracer order");
       VOF_order           = prm.get_integer("VOF order");
       phase_ch_order      = prm.get_integer("phase ch order");
-      potential_ch_order  = prm.get_integer("potential ch order");
+      potential_order     = prm.get_integer("potential ch order");
```

Comment:
The choice was to say that VOF is the default solver for multiphase while Cahn-Hilliard is only an auxiliary solver. All variables/parameters/etc. related to multiphase problems using Cahn-Hilliard have a "ch" added to differentiate them from VOF variables. This may change in the future but it'll stay like this for the moment, except for a major demand for it. In this case, vof variables will have a "vof" prefix added.

Reply:
Seems legit